bundles/XliffBundle/src/ExportDataExtractorService/DataExtractor/DataObjectDataExtractor.php

```diff
@@ -189,6 +189,54 @@ protected function addBlocksInLocalizedfields(Localizedfields $fd, Data $definit
         }
     }
 
+    protected function addBlocksInLocalizedFieldCollections(
+        Data $definition,
+        DataObject\Localizedfield $localizedField,
+        Data $fieldCollectionDefinition,
+        mixed $fieldCollectionItem,
+        AttributeSet $result): void
+    {
+        $locale = str_replace('-', '_', $result->getSourceLanguage());
```

Comment:
can we pass locale from `addLocalizedFieldsInFieldCollections` instead of re-calculating it?

Reply:
just copy/paste from other part in code ^^ but would be nicer, yes. 
im out of time for the next time – is there a chance that you take care of it?

Reply:
sure, I'll take over.